### PR TITLE
May2_may2-sua-file2m2

### PR DESCRIPTION
**Pull request by Vudongmailme**
Please check file2m2, I have just edited it...

### CODE REVIEW

File: file2m2-branch2.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+1
+2
```
Comment (dongvuvd7) [inline file2m2-branch2.txt:2]: line 2?

File: file2m2-branch2.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+3
+4
```
Comment (dongvuvd7) [inline file2m2-branch2.txt:4]: line 4 readed

**Review by dongvuvd7 — COMMENTED** [review]
i have just reviewed it, please check it ...

File: file2m2-branch2.txt (hunk 1 above)
Comment (Vudongmailme) [reply file2m2-branch2.txt:2]: i fixed line 2

File: file2m2-branch2.txt (hunk 2 above)
Comment (Vudongmailme) [reply file2m2-branch2.txt:4]: thanks

**Pull request merged**